src/CellData/DomainContributions.jl

```diff
@@ -1,8 +1,10 @@
+using DataStructures
 
 """
 """
+
```

Comment:
```suggestion
```
No blank line here.  This is supposed to be attaching a [docstring](https://docs.julialang.org/en/v1/manual/documentation/) to the `struct DomainContribution` type.  (The docstring is currently empty as a placeholder, I guess.)